pkg/kubelet/dockertools/manager_test.go

```diff
@@ -1822,21 +1824,20 @@ func TestCheckVersionCompatibility(t *testing.T) {
 		// Invalid apiversion
 		{"invalid_api_version", false},
 	}
-	for i := range apiVersion {
-		apiVersion[i]++
+	for i := range intSlice {
```

Comment:
I have mixed feelings about this test. I know the test tries to be generic, but I find it harder to parse/read. IMO, it's easier to reason with with test cases such as "1.9.1", etc. I know it means that every time we bump the docker version, we'll have to manually change this test, but honestly, I think that's trivial enough and doesn't happen often (we bump docker version not more than twice per release). @Random-Liu, what to do you think?